integration_tests/e2e/prisonerIncentiveLevelDetails.cy.ts

```diff
@@ -0,0 +1,169 @@
+import moment from 'moment'
+import Page from '../pages/page'
+import PrisonerIncentiveLevelDetailsPage from '../pages/prisonerIncentiveLevels/prisonerIncentiveLevelDetailsPage'
+
+const iepSummaryResponse = {
+  bookingId: -1,
+  iepDate: '2017-08-05',
+  iepTime: '2017-08-05T16:04:35',
+  iepLevel: 'Standard',
+  daysSinceReview: 625,
+  nextReviewDate: '2017-08-06',
+  iepDetails: [],
+}
+
+context('Prisoner incentive level details', () => {
+  beforeEach(() => {
+    cy.task('stubFallbackHeaderAndFooter')
+    cy.task('stubNomisUserRolesGetCaseloads')
+    cy.task('stubManageUser')
+    cy.task('stubNomisUserRolesApiUserCaseloads')
+    cy.task('stubPrisonApiLocations')
+    cy.task('stubPrisonIncentiveLevels')
+  })
+
+  context('When user has correct role', () => {
+    beforeEach(() => {
+      const roles = ['ROLE_MAINTAIN_IEP']
+      cy.task('stubSignIn', { roles })
+    })
+
+    it('should display the change incentive level link', () => {
+      cy.task('stubGetIncentiveSummaryForPrisoner')
+      cy.navigateToPrisonerIncentiveLevelDetails().then(result => {
```

Comment:
for ever `cy.navigateToPrisonerIncentiveLevelDetails()` call, you don’t need the `.then(…)` cos the "result" is not being used

Reply:
correct 😓